nil-docs/zkllvm/overview/what-is-zkllvm.mdx

```diff
@@ -20,6 +20,17 @@ The Placeholder system is PLONKish. The circuit components for this system are s
 
 :::
 
+:::tip[Recommended specs]
+
+The following specs are recommended for working with zkLLVM.
+
+- A Unix kernel (an Ubuntu distribution is preferable)
+- At least 64 GBs of RAM
```

Comment:
In my opinion, it's a way too strict advice. I am totally fine with 32 GB, and I believe that it can be reduced. At least for scenario when you don't have to build the compiler and simply install it from the package, even 8 GB should be enough.

Reply:
Oh, God, yes, i missed this one. This is a huge exaggeration! It's pretty hard to find a PC with 64GB RAM :smile:

Reply:
you are both right, and I have amended this part